gatsby-config.js

```diff
@@ -131,7 +131,14 @@ module.exports = {
     `gatsby-transformer-sharp`,
     `gatsby-plugin-image`,
     `gatsby-transformer-yaml`,
-    `gatsby-transformer-garden-svg`,
+    {
+      resolve: 'gatsby-transformer-garden-svg',
+      options: {
+        figmaApiToken: process.env.FIGMA_TOKEN,
+        fileId: '6g87L4FdKZTA3knt3Rsfdx',
```

Comment:
Do we know why this `fileId` does not match the one currently in use within `gatsby-config.js`? As I recall, all website assets should be coming from `'HifWgSrdeTlMMXUrcnrAAH'` ... is that right, @hzhu?

Reply:
I'm not sure if we're going to have to make it an exception because it would be awkward of having a copy of tokens along with their icons inside of `HifWgSrdeTlMMXUrcnrAAH` and would defeat the purpose of having a single source. 

Unless the designers have a way to sync the tokens and the icons inside of the Figma assets file with the current location of the icons.